test/index.test.js

```diff
@@ -57,6 +57,23 @@ test.beforeEach((t) => {
 
 /* Successful scenarios validations */
 
+test.cb('validate retrieving community details, communityUuid provided', (t) => {
```

Comment:
this doesn't look like a high test coverage. pls add tests that go through all the branches of your code.